cleanlab/rank.py

```diff
@@ -111,19 +123,19 @@ def get_self_confidence_for_each_label(
 
     Returns
     -------
-      self_confidence : np.array (float)
-        Return the holdout probability that each example in pred_probs belongs to its
-        label. Assumes pred_probs is computed holdout/out-of-sample."""
+    self_confidence : np.array (float)
+      Return the holdout probability that each example in pred_probs belongs to its
+      label. Assumes pred_probs is computed holdout/out-of-sample."""
 
     # np.mean is used so that this works for multi-labels (list of lists)
-    label_quality_scores = np.array([np.mean(pred_probs[i, l]) for i, l in enumerate(labels)])
-    return label_quality_scores
+    self_confidence = np.array([np.mean(pred_probs[i, l]) for i, l in enumerate(labels)])
+    return self_confidence
 
 
 def get_normalized_margin_for_each_label(
-    labels,
-    pred_probs,
-):
+    labels: np.array,
+    pred_probs: np.array,
+) -> np.array:
     """Returns the "normalized margin" for every example associated pred_probs and
```

Comment:
Can we make it clear: This is a function to compute label-quality scores, where lower scores indicate labels less likely to be correct.

In general, the docstring for every label-quality scoring function should make it clear that this is what it is.

Reply:
Thx good call. Done: 100783d9199110a66737f2d09e2fce762b500c45